README.md

```diff
@@ -167,6 +168,15 @@ slog.Info("a user has logged in",
 )
 ```
 
+### Arguments with forbidden keys
+
+To ensure users don't use reserved words in key-value pairs, you may want to enforce rules to avoid misusages.
+The `forbidden-key` option causes `sloglint` to report usages of forbidden keys. These keys could be typically controlled by slog handlers and its usage should be preserved to avoid key duplication.
```

Comment:
I don't think we should enforce it because by default `slog` doesn't use them - they are used only if you enabled `AddSource` option. 

For everyone that doesn't use them or uses custom handlers, we can break their linter even if they aren't affected by this.